bin/entry.sh

```diff
@@ -5,16 +5,19 @@ WORK_DIR="$(readlink -e "$(dirname "$0")"/..)"
 PATH="${WORK_DIR}/node_modules/.bin:${PATH}"
 cd "${WORK_DIR}"
 
+# shellcheck disable=1090
 test -f "${WORK_DIR}/config/env" && source "${WORK_DIR}/config/env"
 
 service="$(basename "$0")"
 service="${service/balena-/}"
 command="$(command -v node)"
+args=("-r" "source-map-support/register")
```

Comment:
Is this for nicer sentry errors?  It might be better in that case to upload the source maps to sentry and let it handle the mapping rather than paying the cost in production

Reply:
I was actually thinking of nicer local errors, and hadn't even considered Sentry. I can certainly see the argument for "run typescript locally, provide Sentry with source maps" though -- I'll look into it!

Reply:
While getting the source maps into Sentry is definitely the best way forward, it seems to be no small undertaking so I'm going to merge as is for now until we can get Concourse handling Sentry releases properly.

Reply:
Ok, I'm just wondering if this undoes the benefits from not transpiling typescript